sdcm/logcollector.py

```diff
@@ -1204,6 +1204,7 @@ class KubernetesLogCollector(BaseSCTLogCollector):
         FileLog(name='haproxy_ingress.log', search_locally=True),
         FileLog(name='*_cluster_events.log', search_locally=True),
         FileLog(name='kubectl.version', search_locally=True),
+        DirLog(name='test_deploy_quasi_multidc_db_cluster/*', search_locally=True),
```

Comment:
what logs do we have there ? and why do we need test specific collector here ?

Reply:
In the new test we create and delete resources which get cleaned up at the end of the test.
So, we gather narrow set of K8S logs with the namespaces and their resources which are going to be deleted after it.
It is needed for debug. Without gathering of the logs we need either to keep resources around of gather logs at that time. And I chose second variant, because first one makes no sense.

Reply:
Done